uint8 slave_id
uint8 imu_send_data_flag
uint8 dynamixel_ttl_rs485_mixed
uint16 servo_pulley_skip_thresh
spinal/ServoInfo[] servos
